docs/resources/fabric_connection.md

```diff
@@ -731,6 +731,79 @@ resource "equinix_fabric_connection" "fcr2metal" {
 }
 ```
 
+Port to Alibaba Connection EVPL_VC Connection:
+
+### Step by Step Instructions for the Fabric Port to Fabric Alibaba Profile Connection Example Given Below
+
+#### 1. Create and Accept the Connection
+
+Create the connection using the Equinix Terraform provider. Then manually accept the connection request in the Alibaba Portal for the created physical connection.
+
+**Note:** The `connection_name` must follow either one of these patterns:
```

Comment:
I wasn't perfectly clear. We don't need any of the details related to connection name. That's only for when you're integrating with alicloud via the data source